src/cljc/harja/domain/hoitoluokat.cljc

```diff
@@ -59,6 +59,8 @@ Numero on tierekisterin koodi luokalle."}
   [{:nimi "I" :kuvaus "I lk Vilkkaat soratiet" :numero 1 :soratieluo "I lk"}
    {:nimi "II" :kuvaus "II lk Perussoratiet" :numero 2 :soratieluo "II lk"}
    {:nimi "III" :kuvaus "III lk Vähäliikenteiset soratiet" :numero 3 :soratieluo "III lk"}
+   {:nimi "I - uusi" :kuvaus "I lk Vilkkaat soratiet" :numero 4 :soratieluo "I lk - uusi"}
```

Comment:
Aineistoon tuleva teksti tulee olemaan `I lk uusi` tai `II lk uusi`, `:soratieluo` pitäisi vielä vaihtaa täsmäämään näitä. Nämä tosiaan paikkatietopalvelusta saadusta sähköpostista, joka olikin tullut vain minulle.